instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -30,6 +30,16 @@ type gRPCContext struct {
 	record           bool
 }
 
+// AddAttrs adds attributes to the given context.
+func AddAttrs(ctx context.Context, attrs ...attribute.KeyValue) context.Context {
```

Comment:
nit: Consider `WithAttributes` to match the naming pattern for functions that add to the context.